receiver/googlecloudspannerreceiver/internal/metadata/metricsdatapoint_test.go

```diff
@@ -19,32 +19,65 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/model/pdata"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver/internal/datasource"
 )
 
-func TestMetricsMetadata_GroupingKey(t *testing.T) {
-	timestamp := time.Now().UTC()
-	labelValues := allPossibleLabelValues()
-	databaseID := databaseID()
-	metricsDataPoint := &MetricsDataPoint{
-		metricName:  metricName,
-		timestamp:   timestamp,
-		databaseID:  databaseID,
-		labelValues: labelValues,
-		metricValue: allPossibleMetricValues(metricDataType)[0],
-	}
+const (
+	expectedHashValue = "b519d020edc95bf"
```

Comment:
How is this hash generated btw? 

In the event that it is needed to be regenerated, it would be nice to know what to do.

Reply:
I generated it using the same library. Intent is to detect that something changed in library implementation in case we received different value here.

Reply:
Could I ask you to comment the steps on how to do so please as part of this const?

Reply:
Done.